Backpack/Carousel/Classes/CarouselPageViewController.swift

```diff
@@ -45,11 +45,11 @@ final class CarouselPageViewController: UIPageViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        dataSource = self
         delegate = self
     }
     
     func set(images: [UIView], animated: Bool = true) {
+        dataSource = images.count > 1 ? self : nil
         pages.removeAll()
```

Comment:
Disabling scrolling for when there is only not image as mentioned here: https://skyscanner.slack.com/archives/C04NDB73D8S/p1676459461414229?thread_ts=1676046304.364759&cid=C04NDB73D8S